_data/retail.yml

```
@@ -469,3 +572,5 @@ websites:
     img: zooplus.png
     twitter: zooplusDE
     facebook: zooplusde
+    regions:
+      - de
```

```suggestion
    facebook: zooplusde
```
Zooplus serves [24+ countries](https://www.zooplus.com/#js-footer-shops).